x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/logic/integrations/install_ai_prompts.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SECURITY_AI_PROMPTS_PACKAGE_NAME } from '../../../../../../common/detection_engine/constants';
+import type { SecuritySolutionApiRequestHandlerContext } from '../../../../../types';
+import type { ConfigType } from '../../../../../config';
+import { findLatestPackageVersion } from './find_latest_package_version';
+
+export async function installSecurityAiPromptsPackage(
+  config: ConfigType,
+  context: SecuritySolutionApiRequestHandlerContext
+) {
+  try {
+    await findLatestPackageVersion(context, SECURITY_AI_PROMPTS_PACKAGE_NAME);
+  } catch (e) {
+    // fail silently
+    return null;
+  }
+
+  const pkgVersion = await findLatestPackageVersion(context, SECURITY_AI_PROMPTS_PACKAGE_NAME);
+
+  return context.getInternalFleetServices().packages.ensureInstalledPackage({
+    pkgName: SECURITY_AI_PROMPTS_PACKAGE_NAME,
+    pkgVersion,
+  });
```

Comment:
The error should probably be handled like this to avoid a double `findLatestPackageVersion` call when there's no error:

```suggestion
try {
  const pkgVersion = await findLatestPackageVersion(context, SECURITY_AI_PROMPTS_PACKAGE_NAME);

  return context.getInternalFleetServices().packages.ensureInstalledPackage({
    pkgName: SECURITY_AI_PROMPTS_PACKAGE_NAME,
    pkgVersion,
  });
} catch (e) {
  // fail silently
  return null;
}
```

Another good option would be to adjust `findLatestPackageVersion` to return `null` when no package is found, instead of throwing an error. That would help avoid using a clumsy empty catch block, which can backfire in some cases.